tests/kernel/blktests.pm

```diff
@@ -0,0 +1,78 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018-2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Block device layer tests
+# Maintainer: Michael Moese <mmoese@suse.de>, Sebastian Chlad <schlad@suse.de>
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+#use utils 'zypper_call';
+#use power_action_utils 'power_action';
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+
+    #below variable exposes blktests options to the openQA testsuite
+    #definition, so that it allows flexible ways of re-runing the tests
+    my $quick   = get_required_var('blk_QUICK');
```

Comment:
aren't variables in openQA all capital letters by convention?

Reply:
my thinking was that adding  that prefix blk_ would help to see that those vars are local to this pm module. But sure, I will change to adapt to the convention.

Reply:
Something like BLK_ would do